step-definitions/utils/ui.js

```diff
@@ -608,6 +593,22 @@ const goToExceptionPage = async function (exception) {
   await Promise.all([link.click(), this.browser.page.waitForNavigation()]);
 };
 
+const removeYear = async function (field) {
+  const { page } = this.browser;
+
+  const inputField = `input#${convertFieldToHtml(field)}`;
+
+  await page.focus(inputField);
+
+  await page.keyboard.press("Backspace");
+};
+
+const seeError = async function (errorMessage) {
+  const { page } = this.browser;
+
+  await page.$$(`xpath/.//div[@id = "error-message"]//*[text() = "${errorMessage}"]`);
```

Comment:
Same question about there possibly being two editable fields visible at once - this would fail sometimes, in theory, as it'd always reference the first occurring element.